kahuna/public/stylesheets/main.css

```diff
@@ -1523,6 +1523,11 @@ FIXME: what to do with touch devices
     border-left: 1px solid #565656;
 }
 
+.image-details--scroll {
+    height: calc(100vh - 50px - 35px); /* viewport - top-bar - tab-bar */
```

Comment:
Ugh, the tab-bar can be more than 35px (warning messages!)

Might move to CSS grid for this.

![ugh](https://user-images.githubusercontent.com/836140/43323400-776ee99c-91a9-11e8-91d0-63cc2318d37f.gif)